cdap-ui/app/cdap/styles/common.scss

```diff
@@ -49,6 +49,11 @@ body {
     border-collapse: collapse;
     width: 100%;
 
+    &.disabled,
+    &.disabled * {
+      cursor: not-allowed !important;
```

Comment:
The `important` is to enforce for all descendants to have the same cursor (even if they override with specific styles).